src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -411,7 +411,7 @@ CHIP_ERROR AdvertiserMinMdns::Advertise(const OperationalAdvertisingParameters &
 
     FullQName operationalServiceName = AllocateQName("_chip", "_tcp", "local");
     FullQName operationalServerName  = AllocateQName(uniqueName, "_chip", "_tcp", "local");
-    FullQName serverName             = AllocateQName(uniqueName, "local");
+    FullQName serverName             = AllocateQName(params.GetHostname(), "local");
```

Comment:
If you decided to create more generic method for building host name (what's really great), please use it also in the [Discovery_ImplPlatform.cpp](https://github.com/project-chip/connectedhomeip/blob/master/src/lib/mdns/Discovery_ImplPlatform.cpp#L149) to cover `mdns = platform` case and not only the `mdns = minimal` one.

Reply:
That's crucial for Thread devices, as current SRP implementation provides kind of backend for mDNS only in the platform case, so Thread devices are not using minimal mDNS.

Reply:
Can do - I'll take a look today. Thanks.

Reply:
These should now be the same - PTAL.